src/ekat/mpi/CMakeLists.txt

```diff
@@ -1,4 +0,0 @@
-# Add 
```

Comment:
Interesting that we had a stranded cmake files for so long...

Reply:
The possibilities are endless in that line!